modules/ding_webtrekk/ding_webtrekk.module

```diff
@@ -189,12 +192,30 @@ function ding_webtrekk_ding_entity_view($entity, $view_mode) {
   // The page parametes for the ting object view should only be added on the
   // full page view for the material.
   if ($view_mode == 'full' && current_path() == $entity_path) {
+    // Get classification now as we also need it to determine category.
+    $classification = $entity->getClassification();
+
+    // Here category refers to whether the material is fiction/non-fiction. The
+    // only reliable way to determine this, is by using the 'sk' marker from
+    // the DK5 classification.
+    $category = FALSE;
+    // If classification is 'sk' OpenSearchTingObject returns empty string.
+    // See: OpenSearchTingObject::getClassification().
+    if ($classification === '') {
+      $category = 'fiktion';
+    }
+    // Else, if not FALSE, the material has a DK5 classification and is
+    // nonfiction.
+    elseif ($classification !== FALSE) {
+      $category = 'nonfiktion';
+    }
```

Comment:
This is really hacky in many ways. From a very vague description of `getClassification()` to returning an empty string instead of `FALSE` for fiction etc.

I am on the fence regarding whether it makes sense to implement a new method but what might introduce problems of its own. Let's just go with this.

Reply:
Agreed, it's hacky. I also thought about letting `getClassification()` just return the 'sk' marker, but I guess then other code will have to check for this to avoid showing it in frontend.

Reply:
@kasperg I have learned that it's more complicated to determine genre category than by just looking for the 'sk' marker. Some foreign materials can be non-fiction even though they have a classification number and not 'sk'.

I really think we need a method to determine the genre category. I realise that we can't just update the interface, but how about we add it to `OpenSearchTingObject` and make this openseach specific for now?